skl2onnx/shape_calculators/mixture.py

```diff
@@ -0,0 +1,30 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ..common._registration import register_shape_calculator
+from ..common.data_types import FloatTensorType, Int64TensorType
+from ..common.utils import (
+    check_input_and_output_numbers,
+    check_input_and_output_types
+)
+
+
+def calculate_gaussian_mixture_output_shapes(operator):
+    check_input_and_output_numbers(operator, input_count_range=1,
+                                   output_count_range=2)
+    check_input_and_output_types(operator, good_input_types=[FloatTensorType])
```

Comment:
Why is int not allowed as an input type? Scikit allows int features.

Reply:
I hesitate. Statistically, it makes no sense to fix a gaussian mixture on integer data as it cannot be gaussian. I'll fix it.